src/serve/ouroboros/tests.rs

```diff
@@ -86,8 +99,15 @@ async fn test_blockfetch() {
     server.abort();
 }
 
-//#[tokio::test]
-async fn do_chainsync() {
+#[ignore = "broken"]
+#[tokio::test]
+async fn test_chainsync() {
```

Comment:
Don't know why this test won't work. It intersects, requests the block then tries to receive the block, and the server eventually responds with a block, but the client does not see it for some reason and is stuck trying to receive it.